apiary.apib

```diff
@@ -2553,14 +2553,17 @@ Removes a feature assigned to a project template.
 
 # Group SUPER - File Storage Management
 
-Management of file storages. A file storage is represented by the AWS S3 bucket with associated AWS IAM credentials
-with full access to this bucket and access to the AWS STS service.
-At least one file storage has to be registered for one supported region.
+## Management of file storages.
 
+File storage can be pressented:
```

Comment:
presented

Reply:
A nota bene "represented" ;)

Reply:
Nejlíp možná... 

> File storage can either be: 1) Azure Blob storage

Reply:
File storage can be represented ... zůstanu u jednoduchý angličitny.